client/src/constants/form_choices.json

```diff
@@ -0,0 +1,43 @@
+{
+  "concentration": {
+    "main": {
+      "Bullock01": "Bullock (2001) Physical Form",
+      "Bullock01Power": "Bullock (2001) Power-Law",
+      "Duffy08": "Duffy (2008) Power-Law",
+      "Zehavi11": "Zehavi (2011) Power-Law",
+      "Ludlow16": "Ludlow (2016)",
+      "Ludlow16Empirical": "Ludlow (2016) Empirical"
+    },
```

Comment:
I'm fine with extracting these but we should keep our constants in one file. Can these be moved to backend constants?